package.json

```diff
@@ -44,7 +44,7 @@
     "body-parser": "1.18.3",
     "dateformat": "3.0.3",
     "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "logops": "1.0.0",
+    "logops": "1.0.8",
```

Comment:
Should be added to CNR

Reply:
fix in https://github.com/telefonicaid/iotagent-json/pull/352/commits/d667d2183f270698326370b00efd2d63f1e38d33